CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 1.0.0-beta.15 (2017-03-22)
+
+- Images! Images stored in `src/assets` and referenced via `~/assets/` in all HTML/SCSS files are now bundled.
```

Comment:
Why do we have "Images!" at the start? ... Usually we start these entries with a past tense verb. I'd flip this to "Bundled images stored in..." and delete "are now bundled" at the end.